iaf-commons/src/test/java/nl/nn/adapterframework/util/StringUtilTest.java

```diff
@@ -95,4 +95,19 @@ public void testCountRegex() {
 		assertEquals(4, regexCount);
 	}
 
+	@Test
```

Comment:
Make these tests parameterised, perhaps?

Reply:
I tried but it became a bit bloated (using the CsvSource).
One big long annotation wont do the readability any good, so I opted for the 4 assertions instead.

Reply:
Putting each combination on a new line that never bothers me personally but that is also a matter of personal preference of course!